### ISSUE TITLE
is_countable, not Big Endian safe?

### ISSUE THREAD

**Issue by LocutusOfBorg** [symfony/polyfill#134]
Hello, I see a possible endianess issue on is_countable testsuite and latest polyfill release 1.8.0
https://autopkgtest.ubuntu.com/packages/php-symfony-polyfill/cosmic/s390x

```
Setting up autopkgtest-satdep (0) ...
(Reading database ... 73688 files and directories currently installed.)
Removing autopkgtest-satdep (0) ...
autopkgtest [16:09:03]: test command1: phpunit --bootstrap /usr/share/php/Symfony/Polyfill/autoload.php
autopkgtest [16:09:03]: test command1: [-----------------------
PHPUnit 7.2.7 by Sebastian Bergmann and contributors.

SSS............................................................  63 / 766 (  8%)
............................................................... 126 / 766 ( 16%)
............................................................... 189 / 766 ( 24%)
............................................................... 252 / 766 ( 32%)
............................................................... 315 / 766 ( 41%)
..............................................................S 378 / 766 ( 49%)
SS..F.SSS...................................................... 441 / 766 ( 57%)
............................................................... 504 / 766 ( 65%)
............................................................... 567 / 766 ( 74%)
............................................................... 630 / 766 ( 82%)
............................................................... 693 / 766 ( 90%)
............................................................... 756 / 766 ( 98%)
.....SSS..                                                      766 / 766 (100%)

Time: 2.22 seconds, Memory: 18.22MB

There was 1 failure:

1) Symfony\Polyfill\Tests\Php73\Php73Test::testIsCountable
Failed asserting that false is true.

/tmp/autopkgtest.Lf6G0W/build.nrL/src/tests/Php73/Php73Test.php:27

FAILURES!
Tests: 778, Assertions: 726412, Failures: 1, Skipped: 12.
```

I'm tempted to disable this part of the testsuite, since php 7.3 is not yet a real thing, and nothing is using is_countable in the archive, and this is a new test... but can you please have a look, or tell me if this is not a real issue?

the architecture is s390x

**Comment by nicolas-grekas** [comment]
I don't know if intl-resource bundles are endian-safe or not: maybe `tests/Php73/fixtures/en.res` cannot be read on big endian archs? Does this test pass on PHP 7.2? If yes, could it be a PHP bug?

**Comment by LocutusOfBorg** [comment]
Ubuntu has no 7.2 php yet, so we are using 7.2, you can see the version in the build log
https://objectstorage.prodstack4-5.canonical.com/v1/AUTH_77e2ada1e7a84929a74ba3b87153c0ac/autopkgtest-cosmic/cosmic/s390x/p/php-symfony-polyfill/20180723_074320_841de@/log.gz

**Comment by nicolas-grekas** [comment]
Fixed in #140
